server/metrics/metrics.go

```diff
@@ -16,12 +16,31 @@ const (
 	MetricsCloudInstallationLabel = "installationId"
 )
 
+type Metrics interface {
```

Comment:
Looks like multiple PRs are all touching this interface -- lots of merge conflicts ahead!